### DISCUSSION TITLE
AsyncAPI: UserSignedUp Parse Error

### DISCUSSION THREAD

**Discussion by neil-oconnor** [microcks/microcks#810] [Q&A]
Anyone else seeing the parse error thrown by [AsyncAPI Studio](https://studio.asyncapi.com/) when using the [UserSignedUp sample file](https://raw.githubusercontent.com/microcks/microcks/master/samples/UserSignedUpAPI-asyncapi.yml) linked from the Microcks mocking and testing [how-to](https://microcks.io/documentation/using/asyncapi/) page?

Attached a couple of screenshots of what the studio is complaining about. The "examples" are lined up with the "message".

"_Property examples is not expected to be here_"

![UserSignedUp Parse Error view 1](https://user-images.githubusercontent.com/131887648/236204522-c152b482-f5e6-4718-b921-655b5cce68bd.jpg)
![UserSignedUp Parse Error view 2](https://user-images.githubusercontent.com/131887648/236204552-6e6632f5-037a-40d4-9ab5-dd68b287b9d2.jpg)

I did have a look for existing/known problems, the closest I could find is [#254](https://github.com/asyncapi/asyncapi-react/issues/254), but it's not this problem as far as I can tell.

Thanks!

**Comment by lbroudoux** [comment]
Hi @neil-oconnor,

Throughout the years, AsyncAPI spec has evolved (and so as its studio) and some samples from the Microcks code base may still reference older versions of the spec that are no longer supported in the studio... This is usually due to elements that are now mandatory  or examples syntax that has changed. Let us know if some of them are quick fixes so that we can lower the number of warning and error messages.

Cheers,
